plugin/vifm.vim

```diff
@@ -110,22 +110,33 @@ function! s:StartVifm(mods, count, editcmd, ...)
 				let data = b:data
 				buffer #
```

Comment:
Looks like this line also needs to be updated to become
```vim
				if bufnr('%') == bufnr('#') && !data.split
					enew
				else
					buffer #
				endif
```
Otherwise alternative file isn't preserved on:
1. `:e something`
2. `:ene`
3. `:Vifm`
4. `:q` (in Vifm)

Reply:
I'll change and re-test, plus this gives me a few new tests to try

Reply:
This should now be resolved. I repeated all of the tests above and additionally tested preserving alt empty buffers with single-window and multi-window setups. I tested neovim to make sure it wasn't suffering the same issue in any cases.